systemtests/bin/upload_symbols.py

```diff
@@ -103,33 +101,32 @@ def upload_symbols(ctx, expect_code, auth_token, base_url, symbolsfile):
                     )
                 )
 
+                # 403 means the auth token is bad which is not a retryable error
```

Comment:
This section does the same thing, but the code is a little cleaner with fewer if/elif/else blocks to reason about.